platform_versioned_docs/version-23.4.0/enterprise/prerequisites/azure.mdx

```diff
@@ -7,182 +7,310 @@ tags: [azure, prerequisites, configuration]
+:::
 
-To allow ingress on port 8000:
+### Azure Linux VM
 
-1. Enter **Virtual Machines** in the search bar.
+Create a VM instance with these attributes:
+
+- Use **default values** unless otherwise specified.
+- At least **2 CPUS** and **8GB RAM**.
+- **Ubuntu Server 22.04 LTS - Gen2** image.
+- **Accessible by SSH**.
+- Implement DNS or load balancing via [Azure Application Gateway][azure-dns].
+
+Create an Azure Linux VM:
+- [Via the Azure portal][azure-linux-vm-portal]
+- [Via the Azure CLI][azure-linux-vm-cli]
+
+<details>
+  <summary>Create a VM via Azure portal</summary>
+
+  1. Under **Basics**, select your **Subscription** and **Resource group**.
+  1. Under **Instance details**:
+      - Enter a **VM name**
+      - Select the same **Region** as your resource group.
+      - Select the **Ubuntu Server 20.04 LTS - Gen2** image.
+      - Do not set the VM as an **Azure Spot instance**.
+      - Select the **Size** — B2ps v2 or higher is recommended.
+  1. Under **Administrator account**:
+      - Select **SSH public key**
+      - Enter a **username**
+      - Select **Generate new key pair**
+      - Enter a **Key pair name**
+  1. Under **Inbound port rules**:
+      - Select **Allow selected ports**
+      - Select **SSH (22)** and **HTTP (80)** from the dropdown
+  1. Select **Review + create** at the bottom of the page.
+  1. Review your VM details, then select **Create**.
+  1. When the **Generate new key pair** window opens, select **Download private key and create resource**. Your key file will be download as `myKey.pem`. Note the path to which it was downloaded.
+  1. On the page for your new VM, copy the **Public IP address**. 
+
+  To make the VM's IP address static:
+
+  1. Enter **Public IP addresses** in the search.
+  1. Under **Services**, select **Public IP addresses**.
+  1. On the **Public IP addresses** page, select the entry containing your VM name. A page opens with that IP's details.
+  1. Select **Configuration** from the left-hand navigation panel.
+  1. Confirm that your IP address assignment is **Static**.
+  1. Do not add a custom DNS name label to the VM.
+
+  To allow ingress on port 8000:
+
+  1. Enter **Virtual Machines** in the search bar.
+  1. Under **Services**, select **Virtual machines**.
+  1. On the **Virtual machines** page, select your VM name to navigate to the VM details.
+  1. Select **Networking** from the left-hand navigation panel.
+  1. **Add inbound port rule** for port 8000.
+
+  Connect to the VM via SSH:
+
+  1. On a macOS or Linux machine, open a Terminal and set read-only permission on the `myKey.pem` file with `chmod 400 ~/Downloads/myKey.pem`.
```

Comment:
The default app is Terminal on macOS, but on Linux it can be a variety of things, xterm, Konsole, ect. and we have iTerm2 on Mac. I'd just use the generic _terminal_.

```suggestion
  1. On a macOS or Linux machine, open a terminal and set read-only permission on the `myKey.pem` file with `chmod 400 ~/Downloads/myKey.pem`.
```